sdk/keyvault/assets.json

```diff
@@ -2,5 +2,5 @@
   "AssetsRepo": "Azure/azure-sdk-assets",
   "AssetsRepoPrefixPath": "cpp",
   "TagPrefix": "cpp/keyvault",
-  "Tag": "cpp/keyvault_6f97962887"
+  "Tag": "cpp/keyvault_1e79830b40"
```

Comment:
I did a cursory review of this change, to see if any issues pop-up. Looks good :)

https://github.com/Azure/azure-sdk-assets/commit/1e79830b4067eda32566f06288aacdfc9251d924

Reply:
Awesome, thank you